packages/features/bookings/lib/getBookingResponsesSchema.ts

```diff
@@ -9,14 +9,28 @@ type EventType = Parameters<typeof preprocess>[0]["eventType"];
 // eslint-disable-next-line @typescript-eslint/ban-types
 type View = ALL_VIEWS | (string & {});
 
+export const bookingResponse = z.union([
+  z.string(),
+  z.boolean(),
+  z.string().array(),
+  z.object({
+    optionValue: z.string(),
+    value: z.string(),
+  }),
+]);
+
+export const bookingResponsesDbSchema = z.record(bookingResponse);
```

Comment:
It allows disconnecting current event type data from the responses collected. It thus fixes scenarios like
- Earlier, a  field was of type text and has value "Test" and when we changed the type to "checkbox" booking started showing the value as `false`
- Earlier, a field was there and was recorded in a booking but after removing that field/question, it stopped showing in existing bookings as well.